### PR TITLE
Propagate VERBOSE to the inner make script

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: extra/nrnivmodl-core.in [hunk 1]
```diff
@@ -138,8 +138,12 @@ if [ "$params_DEST_DIR" ]; then
     make_params+=("install")
 fi
 
+if [ "$params_VERBOSE" ]; then
+    make_params+=("VERBOSE=1")
+fi
```
Comment (pramodk) [inline extra/nrnivmodl-core.in:143]: simple change that will simplify life, thank you!

**Review by pramodk — APPROVED** [review]

**Comment by alkino** [comment]
Please retest

**Comment by pramodk** [comment]
No idea why sim stack is failing here. (cc: @ferdonline )

**Comment by pramodk** [comment]
Retest this please

**Pull request merged**